docs/SPEC.md

```diff
@@ -1251,11 +1251,9 @@ Intervals has the following parameters:
 | every  | duration                     | Every is the duration between starts of each of the intervals. Defaults to the value of the `period` duration.                                                                                                          |
 | period | duration                     | Period is the length of each interval. It can be negative, indicating the start and stop boundaries are reversed. Defaults to the value of the `every` duration.                                                        |
 | offset | duration                     | Offset is the duration by which to shift the window boundaries. It can be negative, indicating that the offset goes backwards in time. Defaults to 0, which will align window end boundaries with the `every` duration. |
-| filter | (interval: interval) => bool | Filter accepts an interval record and returns a boolean value. Defaults to include all intervals.                                                                                                                       |
```

Comment:
I also dont think that even previous versions of the intervals function used a `filter` param.